src/format.cc

```diff
@@ -22,6 +22,13 @@ template FMT_API auto locale_ref::get<std::locale>() const -> std::locale;
 
 // Explicit instantiations for char.
 
+#if FMT_MSC_VERSION
+// ostream does it implicitly, so we need make it explicitly
+// otherwise we get ODR violation when {fmt} is built as a shared library
+// and fmt/os.h is not included by the user
```

Comment:
ostream -> fmt::ostream to avoid ambiguity with std::ostream

Also let's make it proper sentences:

```
// fmt::ostream does it implicitly, so we need make it explicitly.
// Otherwise we get ODR violation when {fmt} is built as a shared library
// and fmt/os.h is not included by the user.
```

Reply:
thanks